components/media_player/ui/CardTVChannelList.qml

```diff
@@ -0,0 +1,253 @@
+    // FUNCTIONS
+    ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    function onFirstLoadComplete(model) {
+        main.tvchannelListBrowseModel = model.model;
+        obj.browseModelChanged.disconnect(onFirstLoadComplete);
+    }
+
+    function load(tvchannel, type) {
+        swipeView.currentIndex++;
+        if (type === "tvchannellist") {
+            obj.getTVChannelList(tvchannel);
+            obj.browseModelChanged.connect(onBrowseModelChanged);
+        }
+    }
+
+    function onBrowseModelChanged(model) {
+        if (channellistLoader) {
+            if (channellistLoader.source != "qrc:/components/media_player/ui/TVChannelView.qml")
+                channellistLoader.setSource("qrc:/components/media_player/ui/TVChannelView.qml", { "tvchannelModel": model })
+            else if (channellistLoader.item) {
+                channellistLoader.item.tvchannelModel = model;
+                channellistLoader.item.itemFlickable.contentY = 0;
+            }
+        }
+        obj.browseModelChanged.disconnect(onBrowseModelChanged);
+    }
+
+
+    ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    // UI ELEMENTS
+    ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    property alias swipeView: swipeView
+
+    SwipeView {
+        id: swipeView
+        width: parent.width; height: parent.height
+        currentIndex: 0
+        interactive: false
+        clip: true
+
+        Item {
+            property alias tvChannelListListView: tvChannelListListView
+
+            ListView {
+                id: tvChannelListListView
+                width: parent.width; height: parent.height-100
+                spacing: 20
+                anchors { top: parent.top; horizontalCenter: parent.horizontalCenter }
+                maximumFlickVelocity: 6000
+                flickDeceleration: 1000
+                boundsBehavior: Flickable.DragAndOvershootBounds
+                flickableDirection: Flickable.VerticalFlick
+                clip: true
+                cacheBuffer: 3000
+
+                delegate: playListThumbnail
+                model: main.tvchannelListBrowseModel
+
+                ScrollBar.vertical: ScrollBar {
+                    opacity: 0.5
+                }
+
+                header: Component {
+                    Item {
+                        width: parent.width; height: 120
+
+                        Text {
+                            id: title
+                            color: Style.color.text
+                            text: "TV Kanal"//qsTr("My playlists") + translateHandler.emptyString
```

Comment:
Please use `qsTr` to mark this for translation. Default language is English.

Reply:
solved